### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Url Rewrite- Outbound Rules_Screen Reader: In scan mode, focus is going out of the list items at substitution value label

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#146]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #ScreenReader; #Narrator; #Chrome; #NVDA; #MAS1.3.2

**Environment Details:**
URL: https://manage.iis.net/webserver/url-rewrite
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
Screen Reader: Narrator, NVDA
MAS ID Violated: MAS 1.3.2

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Url Rewrite and press enter
6. Tab till outbound rules tab and press enter
7. Tab till Create Rule button and press enter
8. Tab till macros and press enter.
8. Turn on the screen reader in scan mode and verify when navigating through the list items focus is going out of the list items or not

**Actual Result:**
When navigating through the list items focus is going out of the list items 

**Expected Result:**
When navigating through the list items Focus should not go out of the list items 

**Note:**
In Chrome, even if column2 is empty focus is going to column2 but NVDA is just reading as column2 it is not mentioning column2 as empty

**User Impact:**
Disable users may find difficult while working with the application if focus goes out of the list items

Attachments:
[Bug146_outbound rules_focus going out of the list items.pptx](https://github.com/Microsoft/IIS.WebManager/files/2330861/Bug146_outbound.rules_focus.going.out.of.the.list.items.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
